core/preference/preference.go

```diff
@@ -26,17 +26,49 @@ const (
 	TraitInputNumber      TraitInput = "number"
 )
 
+const (
+	// platform default traits
+	PreferenceDisableOrgsOnCreate = "disable_orgs_on_create"
```

Comment:
Instead of prefixing it with Preference, what if we prefix it with the type of trait, that is `Platform`, `Organization`, `User`?